ymap/ymapexport.py

```diff
@@ -116,14 +117,19 @@ def calculate_extents(ymap, obj):
     ymap.streaming_extents_max = get_max_vector(
         ymap.streaming_extents_max, bbmax)
 
+def convert_angle(angle):
+    sine_value = 5 * math.sin(math.radians(angle))
+    cosine_value = 5 * math.cos(math.radians(angle))
+    print(f"3 The second point is ({sine_value},{cosine_value}).")
+    return sine_value, cosine_value
 
 def cargen_from_obj(obj):
     cargen = CarGenerator()
     cargen.position = obj.location
 
-    # TODO: Convert obj rotation to orientX/orientY
-    cargen.orient_x = 0.0
-    cargen.orient_y = 0.0
+    # I have no idea why obj.rotation_euler needs to be multiplied by 57.3 in order to get close to the actual value in Blender, is it because of parental relationship? 
```

Comment:
`57.295 = 180/pi` so with the multiplication you are converting radians to degrees, but then in `convert_angle` you're converting degrees back to radians with `math.radians`.
You can remove this multiplication and the calls to `math.radians`, it should still work since `obj.rotation_euler` is already in radians which `math.sin/cos` expect.